mlflow/utils/validation_common.py

```diff
@@ -0,0 +1 @@
+MAX_TAG_VAL_LENGTH = 5000
```

Comment:
Q: Shall we make this configurable by environment variable ?

Reply:
Not sure, it was moved from current limits here only to avoid circular import error.
I believe for now this be as constant.
If we need env var here - we can use default as current constant and overwrite it only if env var exists.